filetests/isa/x86/windows_systemv_x64_fde.clif

```diff
@@ -0,0 +1,54 @@
+test fde
```

Comment:
:+1: :+1:! I was extremely unsure how to test that #902 generated expected unwind information, other than checking that Lucet could unwind correctly with those patches. This is great, thank you.